tests/unit/job/productionJobHandler.test.ts

```diff
@@ -173,7 +173,7 @@ describe('ProductionJobHandler Tests', () => {
     );
   });
 
-  test('Default production deploy kicks off manifest generation', async () => {
+  test('Default production deploy does not kick off manifest generation', async () => {
```

Comment:
We should change the wording here - by default, we want production deploys (ones enqueued via the slack job) to generate a manifest, we just don't want it to do so in another job 😉 

```suggestion
  test('Default production deploy does not kick off additional manifest generation job', async () => {
```

Reply:
done